lib/content/audio/approaching.ex

```diff
@@ -41,8 +45,13 @@ defmodule Content.Audio.Approaching do
               handle_unknown_destination(audio)
           end
 
-        var ->
+        {var, nil} ->
           {:canned, {"103", [var], :audio_visual}}
+
+        {var, crowding_description} ->
+          {:canned,
+           {"104", [var, Content.Utilities.crowding_description_var(crowding_description)],
+            :audio_visual}}
```

Comment:
Since this is visual as well, presumably the text will be concatenated together. Do we need a space?

Reply:
good question. I don't know for sure and would have to ask Wabtec about that, but we should probably include one for good measure